addons/mail/static/tests/owl/test_utils.js

```diff
@@ -725,11 +728,52 @@ function patchMessagingService(messaging_service, session = {}) {
                 messagingEnv.store.actions._loopFetchPartnerImStatus = () => {};
                 // TODO FIXME `mail_bot` should not have an impact on `mail` tests
                 messagingEnv.store.actions.checkOdoobotRequest = state => state.mailbotHasRequest = false;
+                messagingEnv.store.env.disableAnimation = true;
             },
         },
     });
 }
 
+/**
+ * Create a parent component class which is a strict copy of a given component class.
+ * The main use is to be able to test sub-component removal.
+ * @param componentClass the component class to copy
+ * @param {Object} param1
+ * @param [param1.template] the template of the parent component.
+ *  Default is <{componentClass} prop1="props.prop1" prop2="props.prop2" (...)/>
+ * @param [param1.useStoreFunction] the useStore function of the parent component.
+ *  Default does nothing.
+ * @returns {ParentComponent}
+ */
+function createParentComponentClass(
```

Comment:
I don't think this helper is worth having it: too cryptic, too rigid, hard to read tests, etc...
much better for test to make a component that has intended subcomponent in template and just mount it.